### PR TITLE
[IMP] odoo_sh doc update

### PR DESCRIPTION
**Pull request by sts-odoo**

### CODE REVIEW

File: content/administration/odoo_sh/advanced/scheduled_actions.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+The delay between a scheduled task's next execution date and the actual triggering depends on a lot of parameters and can be delayed according of the state of the execution pool of the shared server.
+
+Scheduled action on staging branches are executed less often than production build.
+
+The platform will deactivate scheduled actions that repeatedly time out.
+
+If the execution time of the scheduled action is business critical then a dedicated server could be a solution.
+On dedicated server the scheduled actions timeout can be modified the project settings.
+
+On the Odoo.sh platform, we cannot guarantee an exact running time for scheduled actions.
```
Comment (jcs-odoo) [inline content/administration/odoo_sh/advanced/scheduled_actions.rst:30]: - list
- phrasing suggestions 
- some English mistakes

```suggestion
- The delay between a scheduled task's next execution date and the actual triggering depends on many
  parameters and can be delayed according to the state of the execution pool of the shared server.
-  Scheduled actions on staging branches are executed less often than production build.
- The platform will deactivate scheduled actions that repeatedly time out.
- If the execution time of a scheduled action is business-critical, then we recommend using a
  dedicated server as the scheduled actions timeout can be modified in the project settings.
- We cannot guarantee an exact running time for scheduled actions on the Odoo.sh platform.
```

File: content/administration/odoo_sh/advanced/submodules.rst [hunk 2]
```diff
@@ -109,3 +109,4 @@ Ignore modules
 If you're adding a repository that contains a lot of modules, you may want to ignore some of them in case there are any
 that are installed automatically. To do so, you can prefix your submodule folder with a :code:`.`. The platform will
 ignore this folder and you can hand pick your modules by creating symlinks to them from another folder.
+This symlink has to be done within your github repository. Do not try to create the symlink wth the container.
```
Comment (jcs-odoo) [inline content/administration/odoo_sh/advanced/submodules.rst:112]: - note
- cap letters GitHub
- typo ("with")

```suggestion

.. note::
   This symlink has to be done within your GitHub repository. Do not try to create the symlink
   with the container.```

File: content/administration/odoo_sh/getting_started/create.rst [hunk 3]
```diff
@@ -79,7 +79,8 @@ Import your database
 ====================
 
 You can import your database in your Odoo.sh project as long as it is in a :doc:`supported version
-</administration/maintain/supported_versions>` of Odoo.
+</administration/maintain/supported_versions>` of Odoo. The branch will automatically switch to the Odoo version of the database.
```
Comment (jcs-odoo) [inline content/administration/odoo_sh/getting_started/create.rst:82]: present tense
https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#grammatical-tenses

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hello @sts-odoo 
Thanks for the pr :)

- please add a redirection in redirects.txt (root folder)
  add this at line 250
  `administration/odoo_sh/advanced/frequent_technical_questions.rst administration/odoo_sh/advanced/scheduled_actions.rst   # frequent_technical_questions -> scheduled_actions`
- make sure to break the line at the 100th character (see [rst guidelines](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character))

File: content/administration/odoo_sh/advanced/scheduled_actions.rst [hunk 4]
```diff
@@ -0,0 +1,30 @@
+
+.. _odoosh-advanced-scheduled_actions:
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply content/administration/odoo_sh/advanced/scheduled_actions.rst:2]: can you apply this as well if possible?
It's not a guideline but it's to remain consistent with all the newly added ref anchors.

File: content/administration/odoo_sh/advanced/scheduled_actions.rst [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+=================
+Scheduled actions
+=================
+
+Special worker
+--------------
+
+On Odoo.sh, cron jobs and incoming mails are handled by a specific
+worker that is not accounted in the workers on your subscription. It is a kind of "hidden free
+worker" bound to every Odoo.sh instance.
```
Comment (jcs-odoo) [inline content/administration/odoo_sh/advanced/scheduled_actions.rst:13]: - 100th character line break. see our [rst guidelines]([url](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character))

```suggestion
On Odoo.sh, cron jobs and incoming mails are handled by a specific worker that is not accounted in
the workers on your subscription. It is a kind of "hidden free worker" bound to every Odoo.sh
instance.
```

File: content/administration/odoo_sh/advanced/scheduled_actions.rst [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+Limitations
+-----------
+
+* The delay between a scheduled task's next execution date and the actual triggering depends on many
+  parameters and can be delayed according to the state of the execution pool of the shared server.
+* Scheduled actions on staging branches are executed less often than production build.
+* The platform will deactivate scheduled actions that repeatedly time out.
+* If the execution time of a scheduled action is business-critical, then we recommend using a
+  dedicated server as the scheduled actions timeout can be modified in the project settings.
+* We cannot guarantee an exact running time for scheduled actions on the Odoo.sh platform.
```
Comment (jcs-odoo) [inline content/administration/odoo_sh/advanced/scheduled_actions.rst:29]: - not a guideline either but we'd rather use `-` for lists, for consistency of the code
- present tense for general truths.

```suggestion
- The delay between a scheduled task's next execution date and the actual triggering depends on many
  parameters and can be delayed according to the state of the execution pool of the shared server.
- Scheduled actions on staging branches are executed less often than production build.
- The platform deactivates scheduled actions that repeatedly time out.
- If the execution time of a scheduled action is business-critical, then we recommend using a
  dedicated server as the scheduled actions timeout can be modified in the project settings.
- We cannot guarantee an exact running time for scheduled actions on the Odoo.sh platform.
```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
There are still a few minor changes required.

Thanks!

**Review by jcs-odoo — APPROVED** [review]

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**